README.md

```diff
@@ -4,13 +4,248 @@ NATS Streaming is an extremely performant, lightweight reliable streaming platfo
 
 [![License MIT](https://img.shields.io/npm/l/express.svg)](http://opensource.org/licenses/MIT) 
 [![Build Status](https://travis-ci.com/nats-io/nats-streaming-server.svg?token=UGjrGa8sFWGQcHSJeAvp&branch=master)](http://travis-ci.com/nats-io/nats-streaming-server)
-[![Coverage Status](https://coveralls.io/repos/github/nats-io/stan-server/badge.svg?branch=master&t=7dcKvx)](https://coveralls.io/github/nats-io/stan-server?branch=master)
-
-[Project Design Document](https://docs.google.com/document/d/1keDwK35YQnOXXKKy2HVV2oOnvEUPFyypT-Tplh8F89c/edit)
+[![Coverage Status](https://coveralls.io/repos/github/nats-io/nats-streaming-server/badge.svg?branch=master&t=7dcKvx)](https://coveralls.io/github/nats-io/nats-streaming-server?branch=master)
 
 NATS Streaming provides the following high-level feature set.
 - Log based.
 - At-Least-Once Delivery model, giving reliable message delivery.
 - Rate matched on a per subscription basis.
 - Replay/Restart
 - Last Value Semantics
+
+### Getting Started
+
+The best way to get the NATS Streaming Server is to use one of the pre-built release binaries which are available for OSX, Linux (x86-64/ARM), Windows. Instructions for using these binaries are on the GitHub releases page.
+
+Of course you can build the latest version of the server from the master branch. The master branch will always build and pass tests, but may not work correctly in your environment. You will first need Go installed on your machine (version 1.5+ is required) to build the NATS server.
+
+### Running
+
+The NATS Streaming Server embeds a NATS Server. Starting the server with no argument will give you a server with sane default settings and a memory based store.
+
+```
+> ./nats-streaming-server
+[73781] 2016/06/20 21:06:52.753188 [INF] Starting nats-streaming-server[test-cluster] version 0.0.1.alpha
+[73781] 2016/06/20 21:06:52.754773 [INF] Starting nats-server version 0.8.2
+[73781] 2016/06/20 21:06:52.754782 [INF] Listening for client connections on localhost:4222
+[73781] 2016/06/20 21:06:52.757134 [INF] Server is ready
+[73781] 2016/06/20 21:06:53.094875 [INF] STAN: Message store is MEMORY
+[73781] 2016/06/20 21:06:53.094887 [INF] STAN: Maximum of 1000000 will be stored
+```
+
+The server will be started and listening for client connections on port 4222 (the default) from all available interfaces. The logs will be displayed to stderr as shown above.
```

Comment:
Suggest adding something like the following to end of this paragraph (even though it may be obvious, best to just state it): "Note that you do not need to start the embedded NATS Server. It is started automatically when you run the NATS Streaming Server."